docs/integrations-guides/rollup-guides/glossary.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Glossary
+
+This glossary contains terms related to rollup integrations and EigenDA. It attemps to use stack-agnostic terms, and detail the equivalent terms in the different rollup stacks.
+
+## Cert Punctuality Window
+
+The time window (in number of L1 blocks) during which a [batcher](#rollup-batcher) must submit a batch to the [rollup inbox](#rollup-inbox) after it has been created.
+
+A cert is considered valid when it is included onchain before the cert's [ReferenceBlockNumber][spec-rbn] (RBN) + the cert's CPW (Cert punctuality window).
+```
+RBN < cert.L1InclusionBlock < RBN+CPW 
+```
+
+A default CPW of 12 hours is recommended. For OP specifically, this number should be at least as large as the [sequencerWindowSize](https://docs.optimism.io/operators/chain-operators/configuration/rollup#sequencerwindowsize).
```

Comment:
`A default CPW of 12 hours is recommended`

What's the block count for 12 hours time? Might make sense to include it here, since CPW is configured in terms of blocks, rather than time.

Reply:
This is a bit tricky because each network has different L1 block times. I added an example number of blocks for ethereum mainnet which has 12sec blocks: [21e57ab](https://github.com/Layr-Labs/eigenda-docs/pull/72/commits/21e57ab47e4b5c8c1da69ee1b9a6f5061aacd9b1)